### PR TITLE
[docs] lintfixes

### PR DESCRIPTION
**Pull request by blefaudeux**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?
Fixes a broken master lint-wise, sorry about that 

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: examples/tutorial_pipe_multiprocess.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 import torch.distributed as dist
 import torch.multiprocessing as mp
 import torch.optim as optim
 
 import fairscale
 from fairscale.nn.model_parallel import initialize_model_parallel
-from helpers import dist_init, getModel, getData, getLossFun
-
 
 DEVICE = "cuda" if torch.cuda.is_available() else "cpu"
```
Comment (Vittorio-Caggiano) [inline examples/tutorial_pipe_multiprocess.py:13]: I would add 
```
RANK = 0  # example
```
here for consistency with the other examples

Comment (Vittorio-Caggiano) [reply examples/tutorial_pipe_multiprocess.py:13]: and below at line 27 
```
device = torch.device("cuda", RANK) if DEVICE == "cuda" else torch.device("cpu")
```

File: examples/helpers.py [hunk 2]
```diff
@@ -19,4 +19,4 @@ def getData(n_batches=1):
 
 
 def getLossFun():
-    return F.nll_loss
\ No newline at end of file
+    return F.nll_loss
```
Comment (min-xu-ai) [inline examples/helpers.py:22]: I see, helpers here. it is not third-party though.

**Review by Vittorio-Caggiano — CHANGES_REQUESTED** [review]
just two nitpicks to make "tutorial_pipe_multiprocess.py" consistent with the other tutorials

**Review by min-xu-ai — APPROVED** [review]
I actually haven't run anything from examples dir myself. How are they supposed to be run? I think we may want to consolidate a bit (not in this PR of course). We have tutorial.rst, examples, unit tests. They all can serve the purpose of examples. Too much overlapping between is probably not efficient for us to maintain. @msbaines

**Review by Vittorio-Caggiano — APPROVED** [review]
lgtm!

**Pull request merged**